lib/Dialect/LWE/IR/LWEAttributes.cpp

```diff
@@ -0,0 +1,61 @@
+#include "include/Dialect/LWE/IR/LWEAttributes.h"
+
+#include "llvm/include/llvm/ADT/SmallSet.h"           // from @llvm-project
+#include "llvm/include/llvm/ADT/StringExtras.h"       // from @llvm-project
+#include "llvm/include/llvm/ADT/StringRef.h"          // from @llvm-project
+#include "llvm/include/llvm/ADT/TypeSwitch.h"         // from @llvm-project
+#include "mlir/include/mlir/Support/LLVM.h"           // from @llvm-project
+#include "mlir/include/mlir/Support/LogicalResult.h"  // from @llvm-project
+
+#define GET_ATTRDEF_CLASSES
+#include "include/Dialect/LWE/IR/LWEAttributes.cpp.inc"
+
+namespace mlir {
+namespace heir {
+namespace lwe {
+
+void LWEParamsAttr::print(AsmPrinter &p) const {
```

Comment:
I recently learned about

` let assemblyFormat = "`<` struct(params) `>`";`

which you might find is sufficient to remove the custom parser/printer

Reply:
I think the issue was that the default assembly format doesn't allow for parsing/printing APInts: 
```
bazel-out/k8-dbg/bin/include/Dialect/LWE/IR/LWEAttributes.cpp.inc:256:52: error: incomplete type 'mlir::FieldParser<llvm::APInt>' used in nested name specifier
  256 |         _result_cmod = ::mlir::FieldParser<APInt>::parse(odsParser);
  ```